envoy/README.md

```diff
@@ -1,25 +1,26 @@
-# Run the Kiosk API with Google Cloud Endpoints
+# Run the Kiosk API with Envoy Proxy
 
-## On your local machine
+Using [gRPC-JSON Transcoding](https://www.envoyproxy.io/docs/envoy/latest/configuration/http_filters/grpc_json_transcoder_filter) the Envoy proxy can publish a JSON REST interface to a gRPC service.
+
+This shows how to do that for the Kiosk API.
 
 ### Prerequisites
 
 *   You have the Kiosk server running.
-*   You have a envoy installed on the same system.
+*   You have Envoy installed on the same system.
 
 ### Steps
 
-All steps should be run from the root directory of this repository.
-
-1.  Rebuild the protos with API descriptors for envoy to consume.
+1.  Rebuild the protos with API descriptors for Envoy to consume.
 
     ```bash
     ./COMPILE_PROTOS.sh
     ```
 
-1.  Start the envoy proxy.
+1.  Start Envoy.
 
     ```bash
     envoy -c envoy.yaml
     ```
 
+1. Access the API with curl or a web browser. For example [this address](http://localhost:8081/v1/kiosks) calls the ListKiosks API method.
```

Comment:
This should be `3.`

Reply:
Markdown [automatically numbers lists](https://stackoverflow.com/questions/12145232/create-an-automatically-numbered-list). 

Verifiable in this case [here](https://github.com/timburks/kiosk/tree/master/envoy).